Lesson-1/assignment/81_黄鑫实_第一课作业

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.4.14;
+
+contract payRollHomework {
+    uint constant payDuration = 10 seconds;
+
+    address owner;     //the employer's address
+       uint salary;    // employee's salary
+    address employee;  //employee's address
+       uint lastPayday;//the last date of payment
+
+    // constructor,once deploy, set owner as as the owner of contract
+    function payRollHomework() {
+        owner = msg.sender;
+        salary = 1;
+    }
+    
+    //udpate the address and salary to the other employee
+    function updateEmployee(address e, uint s) {
+        require(msg.sender == owner);
+        //all of empty addres start by 0x0
+        if(employee != 0x0) {
+            uint payment = salary * (now - lastPayday) / payDuration;
+            employee.transfer(payment);
+        }
+        
+        employee = e;          //update employee's address
+        salary = s * 1 ether;  //update the salary
+        lastPayday = now;      //update the last day to pay salary
+    }
+    
+    //add function to change address
+    function updateAddress(address addr){
+        require(msg.sender == owner);
+        
+        if(employee != 0x0){
+        employee = addr;    
+        }
+    }
+    
+    //add function to change salary
+    function updateSalary(uint sal){
+        require(msg.sender == owner);
+        salary = sal * 1 ether;
```

Comment:
before updating the salary, make sure you pay out the employee with the old salary